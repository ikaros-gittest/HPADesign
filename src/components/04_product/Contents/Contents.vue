<template>
    <v-content>
        <h1>routerView</h1>
    </v-content>
</template>