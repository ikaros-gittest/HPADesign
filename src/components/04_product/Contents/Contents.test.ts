import { mount } from '@vue/test-utils';
import Contents from './Contents.vue';

describe('Contents', () => {
    it('snapshot test', () => {
        const wrapper = mount(Contents);
        expect(wrapper.element).toMatchSnapshot();
    });
});