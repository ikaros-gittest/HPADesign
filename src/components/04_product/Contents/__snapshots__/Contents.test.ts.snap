// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contents snapshot test 1`] = `
<v-content>
  <h1>
    routerView
  </h1>
</v-content>
`;
